### ISSUE TITLE
Inconsistent behavior in scala 2.13

### ISSUE THREAD

**Issue by d1tto** [uuverifiers/ostrich#74]
```
(declare-fun y () String)

(assert (str.in.re (str.replaceall y ">" "&gt;") (re.from_ecma2020 "a")))

(check-sat)
(get-model)
```
The constraint above is unsat in branch scala-2.13, and sat in the master branch.

**Comment by pruemmer** [comment]
Good catch! This is a bug that we fixed on master a few weeks ago, but 
the fix wasn't merged into the scala-2.13 branch yet. This should work 
now, and correctly produce "sat" on both branches.

**Comment by d1tto** [comment]
Thank you very much! I got it.
